documentation/ntfs/ntfs_changelog.md

```diff
@@ -38,4 +38,5 @@
     * Fichier frames.txt rennomé à datasets.txt
     * Attribut frame_id remplacé par dataset_id dans le fichier trips.txt
     * Attribut frame_id supprimé dans le fichier stops.txt
-
+* Version 0.6.2 du 20/06/2016
+    * ajout de l'exention fares pour la gestion des tarifs
```

Comment:
As a good old friend of @antoine-de , maybe the "ancestral master of fares knowledge" could support our fresh new specialist